tests/virt/cluster/aaq/test_aaq_allocation_methods.py

```diff
@@ -8,10 +8,13 @@
 LOGGER = logging.getLogger(__name__)
 TESTS_CLASS_NAME = "TestAAQDifferentAllocationMethods"
 
-pytestmark = pytest.mark.usefixtures(
-    "enabled_aaq_feature_gate_scope_package",
-    "updated_namespace_with_aaq_label",
-)
+pytestmark = [
+    pytest.mark.usefixtures(
+        "enabled_aaq_feature_gate_scope_package",
+        "updated_namespace_with_aaq_label",
+    ),
+    pytest.mark.gating,
```

Comment:
@vsibirsk / @dshchedr can both of you ensure that 1) these tests are 100% stable, 2) release blocker and 3) does not require special hardware

Reply:
AAQ does not require specific hardware and tests look pretty stable (as I know - no any flaky tests). It would be nice to have some of AAQ tests as part of gating group, we encourage customers to use it instead of regular ResourceQuota, so broken AAQ can be a release blocker

 But I'm not sure we need to add all of these test, for saving testing time -  I would keep only: `tests/virt/cluster/aaq/test_arq.py` as a "general" verification of the component.